src/util/cosmos/iscn.js

```diff
@@ -54,8 +54,8 @@ export async function getISCNSigningAddressInfo() {
 }
 
 export function getISCNPrefix(input) {
-  const res = /^(iscn:\/\/likecoin-chain\/[A-Za-z0-9-_]+)\/[0-9]+$/.exec(input);
-  if (!res) return input;
+  const res = /^(iscn:\/\/likecoin-chain\/[A-Za-z0-9-_]+)(?:\/?([0-9]+))?$/.exec(input);
```

Comment:
The `/` is necessary for the group. The group (`(?:\/?([0-9]+))?`) either exist with `/`, or the whole group does not exist.
Current pattern will work because `[A-Za-z0-9-_]+` is greedy and will eat all numbers at the end, leaving nothing for the group. But it's confusing and not robust.

```suggestion
  const res = /^(iscn:\/\/likecoin-chain\/[A-Za-z0-9-_]+)(?:\/([0-9]+))?$/.exec(input);
```

Reply:
hmm what if I want to support tailing slash like `iscn://likecoin-chain/laa5PLHfQO2eIfiPB2-ZnFLQrmSXOgL-NvoxyBTXHvY1/`?

Reply:
`/^(iscn:\/\/likecoin-chain\/[A-Za-z0-9-_]+)(?:\/([0-9]*))?$/`?